netbox_pyswagger/client.py

```diff
@@ -42,21 +47,8 @@ def fetch_swagger_spec(self):
 
         self.api_version = resp.headers['API-Version']
 
-        # Fix for Netbox #1853 https://github.com/digitalocean/netbox/issues/1853
-        for path in spec['paths'].keys():
-            for method in spec['paths'][path].keys():
-
-                if 'parameters' not in spec['paths'][path][method].keys():
-                    continue
-
-                for param in spec['paths'][path][method]['parameters']:
-                    if param['in'] != 'body':
-                        continue
-                    
-                    if 'custom_fields' not in param['schema']['properties'].keys():
-                        continue
-                
-                    if param['schema']['properties']['custom_fields']['type'] == 'string':
-                        param['schema']['properties']['custom_fields']['type'] = 'object'
-                    
+        if self.api_version == "2.2":
```

Comment:
I haven't tried 2.3 but is the issue only present with 2.2 ?

Reply:
good point.  I will update to check for >= "2.2" for now.